docs/projects/reviews/2024/2024-sd-jwt-kotlin-annual.md

```diff
@@ -0,0 +1,39 @@
+# 2024 sd-jwt-kotlin (SD-JWT Implementation)
+
+## Project Health
+
+The main focus of this project was to provide a Kotlin implementation for the
+IETF SD-JWT draft. However, the project remain without any updates for a while and
+the maintainer decided to post an announcement stating this project is no longer maintained.
+However during TAC meeting for annual review, there was a volunteer stepped up to 
+take care of this project.
+
+
+## Maintainer Diversity
+
+As before, the project is maintained by a single maintainer.
+It's expected to have a single maintainer for the next year as well.
+
+## Project Adoption
+
+This project was used by German Bundesdruckerei but not sure they are still using.
+
+## Goals
+
+### Performance Against Prior Goals
+
+The project implemented with success to it's initial goal but not well maintained or used.
+
+### Next Year's Goals
+
+This project aims to have a cloned copy of eudi-reference-wallet-sdjwt to have every language
+implementation stays under OWF umbrella.
+
+## Help Required
+
+The project is maintainable with one maintainer and no further aid is needed.
+We are planning to automate the process (maybe using github actions).
```

Comment:
Might be worth adding this as a goal for next year (although, at the same time, you may be asking for help from the community for automating).

Reply:
Goal